### DISCUSSION TITLE
[질문]Naver Clova Service Retrofit 관련 질문

### DISCUSSION THREAD

**Discussion by ChaMinXD** [TEAM-DAME/Dame-Android#92] [2. Q&A] [answered]
![image](https://user-images.githubusercontent.com/96709529/202904283-72681494-5107-47e3-8b3f-8561d4a18067.png)

위에 NETWORK_STATS_SERVICE: String 가 어떤 값인가요 ? 원래 post api 쏠때 content 만 쏘면되는걸로알고있는데 해당 string 에 어떤식으로 값을 주어야할까요 ?

**Comment by onseok** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/76798309/203553333-d75a7a52-cc68-4f1f-976e-31daa9abd288.png)

원래 처음 해당 부분 설계를 했을 때에는 위의 사진처럼 코드를 구현했습니다.
request body 타입 부분도 코드가 조금 다른 것 같습니다.
민혁님 말대로 request body에 `SentimentAnalyzeRequest ` 타입의 데이터만 서버에 보내면 됩니다.
`NETWORK_STATS_SERVICE` 저 부분은 코드를 구현하면서 자동으로 import 된 것 같은데, 저 부분이 없어야 하는 것이 맞는 것 같습니다.
 최신화 후 다시 한번 시도 부탁드립니다!

**Reply by ChaMinXD** [reply]
확인했습니다 !
